File: ProjectTrackercs/www/PTWebService/App_Code/ResourceAssignmentData.cs
using System;

public class ResourceAssignmentData
{
  private Guid _projectId;
  private string _projectName;
  private string _assigned;
  private int _role;

  public Guid ProjectId
  {
    get { return _projectId; }
    set { _projectId = value; }
  }

  public string ProjectName
  {
    get { return _projectName; }
    set { _projectName = value; }
  }

  public string Assigned
  {
    get { return _assigned; }
    set { _assigned = value; }
  }

  public int Role
  {
    get { return _role; }
    set { _role = value; }
  }
}

File: samples/CslaNet/vb/ProjectTrackervb/www/PTWebService/App_Code/ResourceAssignmentData.vb
Option Strict On

Imports Microsoft.VisualBasic

Public Class ResourceAssignmentData

  Private _projectId As Guid
  Private _projectName As String
  Private _assigned As String
  Private _role As Integer

  Public Property ProjectId() As Guid
    Get
      Return _projectId
    End Get
    Set(ByVal value As Guid)
      _projectId = value
    End Set
  End Property

  Public Property ProjectName() As String
    Get
      Return _projectName
    End Get
    Set(ByVal value As String)
      _projectName = value
    End Set
  End Property

  Public Property Assigned() As String
    Get
      Return _assigned
    End Get
    Set(ByVal value As String)
      _assigned = value
    End Set
  End Property

  Public Property Role() As Integer
    Get
      Return _role
    End Get
    Set(ByVal value As Integer)
      _role = value
    End Set
  End Property

End Class
